src/generate-note-controller.js

```diff
@@ -51,6 +51,7 @@ module.exports = function(scribe){
       //scribe command events
       emitter.on('command:note', tag => this.note(tag));
       emitter.on('command:toggle:single-note', tag => this.toggleSelectedNotesCollapseState(tag));
+      this.ensureNoteIntegrity();
```

Comment:
Could we just have a comment about why we do this? So we don't accidentally remove it in the future.

And even better, if we could have a test as well that checks for them being present before an edit is made. Not a blocker though, fine to make an issue for adding the test.